src/shadow.class.js

```diff
@@ -162,6 +163,9 @@
       if (this.offsetY !== proto.offsetY) {
         obj.offsetY = this.offsetY;
       }
+      if (this.affectStroke !== proto.affectStroke) {
+        obj.affectStroke = this.affectStroke;
+      }
       return obj;
```

Comment:
That we have to remove some tests as well, and that we have to wait for 2.0 i think.
I would merge it and make a checklist for 2.0 to remove some old things.